client/question/question.js

```diff
@@ -562,3 +572,11 @@ tie.constant('DEFAULT_AUTOSAVE_SECONDS', 5);
  * @constant
  */
 tie.constant('DISPLAY_AUTOSAVE_TEXT_SECONDS', 1);
+/**
+ * Question Id of the question in case there isn't a question id
```

Comment:
Default question ID to use if no qid parameter is specified in the URL.

Reply:
Done.